judging/scripts/generate_predictions.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
This is the wrapper script that `cron` runs. It is incomplete -- note the TODOs.